docs/velox-backend-limitations.md

```diff
@@ -127,7 +138,7 @@ Velox does not create table as select, e.g.,
 ```scala
 spark.range(100).toDF("id")
   .write
-  .format("velox")
+  .format("parquet")
   .saveAsTable("velox_ctas")
 ```
 
```

Comment:
can you please also add one example on the working case(static partition write)?